sqswatcher/sqswatcher.py

```diff
@@ -206,6 +208,29 @@ def pollQueue(scheduler, q, t, proxy_config):
 
         time.sleep(30)
 
+
+def getInstanceMappingFile(region, proxy_config):
+    s3 = boto3.resource('s3', config=proxy_config)
+
+    try:
+        if not os.path.exists('/opt/cfncluster/'):
+            os.makedirs('/opt/cfncluster/')
+    except OSError as ex:
+        if ex.errno == errno.EEXIST and os.path.exists('/opt/cfncluster/'):
+            pass
+        else:
+            log.critical('Could not save instance mapping file. Failed with exception: %s' % ex)
+            raise
+
+    bucket_name = region + '-cfncluster'
+    try:
+        bucket = s3.Bucket(bucket_name)
+        bucket.download_file('instances/instances.json', '/opt/cfncluster/instances.json')
+    except ClientError as e:
+        log.critical("Could not save instance mapping file. Failed with exception: %s" % e)
```

Comment:
Just more context, "Could not save instance mapping file %s from S3 bucket %s. ..."